.vscode/launch.json

```diff
@@ -7,22 +7,29 @@
             "type": "extensionHost",
             "request": "launch",
             "runtimeExecutable": "${execPath}",
-            "args": ["--extensionDevelopmentPath=${workspaceRoot}" ],
-            "stopOnEntry": false,
-            "sourceMaps": true,
-            "outFiles": [ "${workspaceRoot}/out/src/**/*.js" ],
-            "preLaunchTask": "npm"
+            "args": [
+                "--extensionDevelopmentPath=${workspaceFolder}"
+            ],
+            "outFiles": [
+                "${workspaceFolder}/dist/**/*.js"
+            ],
+            "preLaunchTask": "npm: webpack"
```

Comment:
webpack or webpack-dev?

Reply:
`webpack-dev` will never return so the extension host cannot start.

Reply:
so what is webpack-dev used for?

Reply:
It's copied from the sample, and not used currently.